scripts/ci-is-vendor-in-sync.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash -e
+#!/bin/bash
```

Comment:
Is this the right way to add a new step to CI?

Reply:
You have to configure prow to run some sort of command. Right now it runs the three scripts for CI as three different tests. We can either augment an existing script or add a new script (and configure a new presumbit).

Reply:
I don't really have an opinion on augment existing script vs add a new script + configure a new presubmit - Any preference from your side?

Reply:
+1 to a `new script + configure a new presubmit`

Reply:
Fine with me. Can you tell me where a new presubmit gets configured?